src/nmodl/modl.cpp

```diff
@@ -224,12 +224,7 @@ no longer adequate for saying we can not */
 #endif
  if (nmodl_text) {
 	Item* q;
-	char* pf = NULL;
-#if HAVE_REALPATH
-	pf = realpath(finname, NULL);
-#endif
-	fprintf(fcout, "\n#if NMODL_TEXT\nstatic const char* nmodl_filename = \"%s\";\nstatic const char* nmodl_file_text = \n", pf ? pf : finname);
-	if (pf) { free(pf); }
+	fprintf(fcout, "\n#if NMODL_TEXT\nstatic const char* nmodl_filename = \"%s\";\nstatic const char* nmodl_file_text = \n", finname);
 	ITERATE(q, filetxtlist) {
```

Comment:
@nrnhines : is / was there any usefulness of having full absolute path in the translated cpp file?

Reply:
Translated mod files contain the full text of the mod file. The absolute path on translation conceivably can aid the human reader in knowing the provenance.

Reply:
I added a new `NRN_AVOID_ABSOLUTE_PATHS` option (default: false) that now hides this change. We can use that in the CI config.

For `ivoc_help` I copied the logic of `nmodl_filename = ...` as this seemed neater.